### DISCUSSION TITLE
Lesson 8 Challenge - Could not understand how to approach this challenge.

### DISCUSSION THREAD

**Discussion by subhradeepsamantray** [Cyfrin/foundry-full-course-cu#755] [Q&A] [answered]
Hello All,

I am having difficulties in understanding how to approach this challenge where I am getting an IPFS link, I don't know how to proceed in this. Any help / hints to proceed in this challenge will be appreciated.
Also I have another doubt, I have seen in other discussions for this challenge like #102 , people are able to run foundry cast commands to solve this challenge. How are you able to interact with the contract through commands ? Do we need to download the contracts from Etherscan and then run the cast commands or we can directly run the commands in our terminal ??? Please help!

Thanks,
Subhradeep Samantray

**Comment by subhradeepsamantray** [comment]
I have found the website using IPFS HTTP gateway. But don't know what to do next. Can anyone help with the next steps?

**Reply by Niferu** [reply]
Hi @subhradeepsamantray 

Try checking console errors ;)

**Reply by subhradeepsamantray** [reply, marked as answer]
Hey!, Thanks for the help!, I used the console errors and the hints in other discussions , particularly #102 and solved the challenge.
